### PR TITLE
Setup gradle project

### LINKED ISSUE #2: Gradle setup

### PR DESCRIPTION
**Pull request by nicolasfara**
Setup the project with gradle as build tool.

Closes #2

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    id("com.gradle.enterprise") version "3.10"
+}
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
+        publishOnFailure()
+    }
+}
```
Comment (ndido98) [inline settings.gradle.kts:11]: You probably forgot a trailing newline

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.scala-lang:scala3-library_3:3.1.2")
+    testImplementation("org.scalatest:scalatest_3:3.2.11")
+    testRuntimeOnly("org.junit.platform:junit-platform-engine:1.8.2")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher:1.8.2")
+    testRuntimeOnly("co.helmethair:scalatest-junit-runner:0.1.10")
```
Comment (ndido98) [inline build.gradle.kts:23]: Did you try configuring the test task to use the ScalaTest runner instead of the JUnit one?

Comment (nicolasfara) [reply build.gradle.kts:23]: JUnit is mandatory, but I successfully configured the test suite to use scalatest, so no annotation is required for each test class.

File: settings.gradle.kts (hunk 1 above)
Comment (nicolasfara) [reply settings.gradle.kts:11]: Yes... I'll fix

**Review by vitlinda — APPROVED** [review]

**Review by giacomocavalieri — APPROVED** [review]

**Review by giacomocavalieri — APPROVED** [review]

**Pull request merged**